lib/ddtrace/contrib/http/ext.rb

```diff
@@ -11,6 +11,7 @@ module Ext
         ENV_ANALYTICS_SAMPLE_RATE_OLD = 'DD_HTTP_ANALYTICS_SAMPLE_RATE'.freeze
         SERVICE_NAME = 'net/http'.freeze
         SPAN_REQUEST = 'http.request'.freeze
+        CONNECTION_ERROR_RESOURCE = 'UNKNOWN'.freeze
```

Comment:
The connection error happens before the user call even calls the `request` (in case of not using the shortcut methods provided by `Net::HTTP`), we don't know what kind of method it is.

Feel free to suggest better resource name for the connection errors spans - this will only be present on error spans